modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/resource/v2_0/DataDefinitionResourceImpl.java

```diff
@@ -416,13 +416,16 @@ public DataDefinition postSiteDataDefinitionByContentType(
 		DDMForm ddmForm = DataDefinitionDDMFormUtil.toDDMForm(
 			dataDefinition, _ddmFormFieldTypeServicesTracker);
 
+		DataDefinitionContentType dataDefinitionContentType =
+			_dataDefinitionContentTypeTracker.getDataDefinitionContentType(
+				contentType);
+
+		ddmForm.setAllowInvalidAvailableLocalesForProperty(
+			dataDefinitionContentType.allowEmptyDataDefinition());
```

Comment:
I think you should call `dataDefinitionContentType.allowInvalidAvailableLocalesForProperty` , rigth?

Reply:
Right, @jeyvison. Sorry 😅
I forced a push for this pull with the fix. Thanks!